packages/backend/src/graphql/resolvers/wallet_address.test.ts

```diff
@@ -134,75 +133,68 @@ describe('Wallet Address Resolvers', (): void => {
       }
     )
 
-    test.each`
-      publicName
-      ${'Bob'}
-      ${undefined}
-    `(
-      'Can create a wallet address with additional properties (publicName: $publicName)',
-      async ({ publicName }): Promise<void> => {
-        input.publicName = publicName
-        input.additionalProperties = [
-          { key: '', value: '', visibleInOpenPayments: false },
-          { key: 'key', value: '', visibleInOpenPayments: false },
-          { key: '', value: 'val', visibleInOpenPayments: false },
-          { key: 'key', value: 'val', visibleInOpenPayments: false },
-          { key: 'key-public', value: 'val', visibleInOpenPayments: true }
-        ]
-        const response = await appContainer.apolloClient
-          .mutate({
-            mutation: gql`
-              mutation CreateWalletAddress($input: CreateWalletAddressInput!) {
-                createWalletAddress(input: $input) {
-                  code
-                  success
-                  message
-                  walletAddress {
-                    id
-                    asset {
-                      code
-                      scale
-                    }
-                    url
-                    publicName
+    test('Can create a wallet address with additional properties', async (): Promise<void> => {
```

Comment:
we should check the correct properties were returned in the response

Reply:
updated [f73e66d](https://github.com/interledger/rafiki/pull/2769/commits/f73e66d5a8bbf554e965b29a664c90993181caf5)